conf.d/rewrite.conf

```
@@ -44,3 +44,7 @@ rewrite /tutorial/vue/(.*) /developing/vue-tutorial/\$1 permanent;
 # Data-visualization
 rewrite /data-visualization/basic-charts/ /data-visualization/simple-charts/ permanent;
 rewrite /data-visualization/advanced-charts/ /data-visualization/complex-charts/ permanent;
+
+#v11
+rewrite https://v11.carbondesignsystem.com/migrating/guide/\$1 /migrating/guide/\$1 permanent;
```

Quick question, would this also work?

```suggestion
rewrite /migrating/guide/\$1 /migrating/guide/\$1 permanent;
```

---

I’m actually not even sure how to text it. Since v11 is deployed. I’m trying to redirect from v11.carbondesignsystem.com to regular carbondesignsystem.com